src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/readme.md

```diff
@@ -4,95 +4,364 @@ Reference documentation for the current version of the REST API supported by the
 
 :warning: **This is draft documentation for the next version of the SES REST API** - see the [previous version](rest-api-2017-05-01.5.0.md) for the current release.
```

Comment:
We don't need to copy the old content here into a previous version, because that version (with vmid removed and expiry added) was never released - is that right?

Reply:
Yes, that's correct.